components/tyk-dev-portal/README.md

```diff
@@ -0,0 +1,174 @@
+```
+    
+
+This removes all the Kubernetes resources associated with the chart and deletes the release.
+
+## Upgrading Chart
+```bash
+helm upgrade tyk-dev-portal tyk-helm/tyk-dev-portal -n tyk
+```
+
+## Configuration
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To get all configurable options with detailed comments:
+```bash
+helm show values tyk-helm/tyk-tyk-dev-portal > values.yaml 
+```
+    
+You can update any value in your local values.yaml file and use `-f [filename]` flag to override default values during installation. Alternatively, you can use `--set` flag to set it during chart installation.
+
+### Tyk Developer Enterprise Portal License (Required)
+
+Tyk Developer Enterprise Portal License is required. It can be set up in `license` or through secret `useSecretName`. The secret should contain a key called `DevPortalLicense`.
+
+```yaml
+  # Developer Portal license.
+  license: ""
+```
+
+### Storage Settings
+
+Tyk Enterprise Portal supports different storage options for storing the portal's CMS assets such as images, theme files and Open API Specification files. Please see the [Enterprise Portal Storage settings](https://tyk.io/docs/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration#portal-settings) page for all the available options. Helm chart supports the setting of the following fields in `storage` section:
+
+```yaml
+storage:
+  # Configuration values for using an SQL database as storage for Tyk Developer Portal
+  # In case you want to provide the connection string via secrets please
+  # refer to the existing secret inside the helmchart or the
+  # .Values.global.secrets.useSecretName variable
+  # User can set the storage type for portal.
+  # Supported types: fs, s3, db
+  type: "db"
+  # Configuration values for using s3 as storage for Tyk Developer Portal
+  # In case you want to provide the key ID and access key via secrets please
+  # refer to the existing secret inside the helm chart or the
+  # .Values.useSecretName field and a secret containing
+  # the keys DevPortalAwsAccessKeyId and respectively,
+  # DevPortalAwsSecretAccessKey
+  s3:
+    awsAccessKeyid: your-access-key
+    awsSecretAccessKey: your-secret-key
+    region: sa-east-1
+    endpoint: your-portal-bucket
+    bucket: https://s3.sa-east-1.amazonaws.com
+    acl: private
+    presign_urls: true
+  persistence:
+    # User can mount existing PVC to enterprise portal
+    # Make sure to change the kind to Deployment if you are mounting existing PVC 
+    mountExistingPVC: ""
+    storageClass: ""
+    accessModes:
+      - ReadWriteOnce
+    size: 8Gi
+    annotations: {}
+    labels: {}
+    selector: {}
+```
+
+### Database Settings
+Portal uses database to store metadata related to the portal, such as API products, plans, developers, applications, and more.
+Please refer to this page for more details.
```

Comment:
Can you please link the page here

Reply:
Ohh sorry. I forgot.
Resolved in [72084ca](https://github.com/TykTechnologies/tyk-charts/pull/197/commits/72084ca40a049655520207b89c7888f407b333e6)